framework/configuration/rest-resource.rst

```diff
@@ -20,9 +20,9 @@ resources.
 
 The steps to implement a REST resource in any module:
     #. Add the required Maven dependency
-    #. Extend the Java interface :java:ref:`ch.tocco.nice2.rest.core.spi.RestResource` and define
+    #. Extend the Java interface :abbr:`ch.tocco.nice2.rest.core.spi.RestResource (RestResource)` and define
```

Comment:
:java:ref: wurden falsch ersetzt in diesem File (der volle Klassenname sollte als Popup angzeigt werden, nicht umgekehert)